### ISSUE TITLE
Form fields in the wrong place

### ISSUE THREAD

**Issue by tiredelk** [danfickle/openhtmltopdf#252]
Hi, 

I'm converting a very simple html with a form (input text, textarea, checkbox  and radio buttons) into PDF. The problem is the position of the fillable boxes in the final PDF, they are all to the left/top.

I tried to fix it with css but it doesn't seem to apply to the fillable boxes (or I don't know how to).

[form.html.txt](https://github.com/danfickle/openhtmltopdf/files/2193929/form.html.txt) - [form.pdf](https://github.com/danfickle/openhtmltopdf/files/2193928/form.pdf)

Any suggestions?

**Comment by tiredelk** [comment]
As sugested by @rototor the style `@page { margin: 0.01mm }` is a work around. 

But if the content exceeds the first page then the fillable boxes of the second (I haven tested with more than 2 pages) go under the second page, out of it.


[form-long.html.txt](https://github.com/danfickle/openhtmltopdf/files/2210465/form-long.html.txt)
[print.css.txt](https://github.com/danfickle/openhtmltopdf/files/2210468/print.css.txt)
[form-long-ohtp.pdf](https://github.com/danfickle/openhtmltopdf/files/2210466/form-long-ohtp.pdf)

**Comment by danfickle** [comment]
Both the page margin issue and the multiple pages issue were caused by failing to clone the page transform when we passed it to the form object to process later. Meaning that when we came to the end and processed all forms/controls we used the last page transform for each instead of the correct one.

Anyway, thanks for reporting this @tiredelk and both examples now work. Let us know if you find any other issues with form handling.
